packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -409,43 +411,36 @@ $Calendar-dayPicker-margin: 10px;
     }
   }
 
-  // Show the day picker and month picker side-by-side, if enabled
-  .pickerIsMonthPickerVisible {
+  // Without modifying the Pickadate JS, this transparent
+  // button is necessary to toggle the month view.
+  .headerToggleView {
+    height: $Calendar-day;
+    @include ms-left(70px);
+  }
+
+  // Show month picker, if enabled
+  .monthPickerVisible {
     // Swap margin for padding so that the border extends the full height.
     .dayPicker {
       margin: -$Calendar-dayPicker-margin 0;
       padding: $Calendar-dayPicker-margin 0;
     }
+
     // Contains the calendar view for picking a day.
     .dayPicker {
       @include ms-borderBox;
       @include border-right(1px, solid, $ms-color-neutralLight);
-      width: 220px;
+      width: 212px;
```

Comment:
The daypicker looks to be 212px in the comps
![2](https://user-images.githubusercontent.com/25463697/29686552-379befca-88ce-11e7-9b40-2f86e0d203b2.jpeg)